src/style.css

```diff
@@ -1,3 +1,77 @@
+:root {
+  --nav-color: #00acdc;
+  --link-height: 60px;
+}
+
+* {
```

Comment:
Avoid using `*` for styling. Just move this to body, font style will cascade to child components